src/App.tsx

```diff
@@ -1,23 +1,20 @@
-import { Home, Plan, Discover, Check, NoPage } from "./Pages";
-import { Navbar } from "./componets";
+import { Home, Plan, Discover, Check, NoPage, Kitchen } from "./Pages";
```

Comment:
Optional but good to know about:

As a little exercise, it would be good to implement lazy loading for these route pages, so we're only loading them as needed and reducing the initial bundle required to load the app and only requesting & loading pages when they're rendered, IE when a route is visited.

- Check out the docs [here](https://legacy.reactjs.org/docs/code-splitting.html#reactlazy)

Shouldn't be a huge change but don't stress if it feels like too much.

Reply:
I've made the changes that you recommended and did some research to the questions you gave me as well.

When would lazy loading be best to apply?
When you have a lot of page’s/routes to load on website. Using lazy loading can help with performance by only loading what is necessary

What are the negatives of lazy loading?
Will require more coding, older browsers and specific environments might not be compatible,

Is there any risk?
UX if user has a slow network resulting in delayed loading of components, can also affect accessibility if components containing important info are lazy loaded.

Should they always be used or just some of the time?
Lazy loading should be used in bigger projects and complex components. Not suitable for small projects